### ISSUE TITLE
Research where localization configuraiton files will live and what the workflow will be for creating them.

### ISSUE THREAD

**Issue by hoosteeno** [mozilla/bedrock#7605]

**Comment by hoosteeno** [comment]
Live in the bedrock repo; mirrored in the fluent repo. 

@pmac Can you ping Axel to get more details here?

**Comment by pmac** [comment]
@Pike ^^

**Comment by pmac** [comment]
What I meant by "bedrock settings" was that currently our production list of locales is hard coded here:

https://github.com/mozilla/bedrock/blob/786ca831226ea79c4c5de20bfe17e0664958aff6/bedrock/settings/base.py#L109-L122

We could just as easily load that from a config file in the fluent repo or bedrock fluent directory.

**Comment by Pike** [comment]
Yeah. We have projects that distinguish between "stage-only" or "nightly" locales, and release-ready ones.

But that might be part of our other conversation around automated production deployment.

What I expect is that we'll have an `l10n.toml` in the top directory, like we do in https://github.com/mozilla-mobile/fenix/blob/master/l10n.toml for example.
